.github/workflows/pr-lint.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         with:
           types: |
             chore
+            ci
```

Comment:
The schema we follow does not use `ci` as type. We use `ci` as scope for the change (i.e. `chore(ci): ...`

Please remove this change

Reply:
As far as i can tell this change fits under 5 categories: chore, fix, docs, refactor, and feature.

But whatever.